### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mayankgrover**
player implemented

### CODE REVIEW

**Comment by ADARSHKUMAR2** [comment]
Good code

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # BattleGame
 master branch 
+
+feature-player
```
Comment (mayankgrover) [inline README.md:4]: asdfasdf

**Review by ADARSHKUMAR2 — CHANGES_REQUESTED** [review]
OK

**Review by mayankgrover — COMMENTED** [review]
asdfafa

**Review by ADARSHKUMAR2 — APPROVED** [review]
Approved

**Pull request merged**